src/conf_mode/container.py

```diff
@@ -418,6 +418,23 @@ def generate_run_arguments(name, container_config):
         entrypoint = json_write(container_config['entrypoint'].split()).replace('"', "&quot;")
         entrypoint = f'--entrypoint &apos;{entrypoint}&apos;'
 
+    healthcheck = ''
+    if 'health_check' in container_config:
+      if 'command' in container_config['health_check']:
+        health_cmd = container_config['health_check']['command']
+        healthcheck += f' --health-cmd="{health_cmd}"'
+      if 'interval' in container_config['health_check']:
+        health_int = container_config['health_check']['interval']
+        healthcheck += f' --health-interval={health_int}'
+      if 'timeout' in container_config['health_check']:
+        health_to = container_config['health_check']['timeout']
+        healthcheck += f' --health-timeout={health_to}'
+      if 'retries' in container_config['health_check']:
+        health_rt = container_config['health_check']['retries']
+        healthcheck += f' --health-retries={health_rt}'
+    else:
+      healthcheck = ' --no-healthcheck'
```

Comment:
Not quite sure about that one

Old behaviour disabled health checks defined in the container when container networking was used, but not when host networking was used (looks like kind of an oversight).

With this change, health checks would be disabled in both cases unless this is set:
```
set container name sample health-check
```

Alternatively I could make it explicit and only disable health-check when:
```
set container name test health-check disable
```